### ISSUE TITLE
Deploy to pypi

### ISSUE THREAD

**Issue by mahenzon** [AdCombo/flask-combo-jsonapi#11]
This issue is blocked by #10

- [ ] start with version 1.0
- [ ] Setup Github Action for this task.
- [ ] Update installation manual https://github.com/AdCombo/flask-combo-jsonapi/blob/develop/docs/installation.rst/

**Comment by methuselah-0** [comment]
Any progress on this? Here's a workflow you can use:
https://github.com/actions/starter-workflows/blob/main/ci/python-publish.yml

This workflow will upload a Python Package using Twine when a release is created
For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries

**Comment by mahenzon** [comment]
@methuselah-0 PR welcomed 😀

**Comment by methuselah-0** [comment]
The thing is that the workflow needs you to add twine user-credentials, and I think those should be yours, not mine.

If you look in this link:
https://docs.github.com/en/free-pro-team@latest/actions/guides/building-and-testing-python#publishing-to-package-registries

there are these 2 lines:
    - name: Build and publish
      env:
        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}

and a link to this page: https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository

Otherwise, I would probably create a PR for you.

Basically, all you need to do is to add the linked action to .github/workflows/main.yml and create the secrets as described in the link.

**Comment by mahenzon** [comment]
We can add token to secrets
I know how to use twine 
https://github.com/mahenzon/aioalice/blob/master/.github/workflows/publish_to_pypi.yml

**Comment by methuselah-0** [comment]
Thanks for merging!

The new github-action worked and the package is here(not searchable on pypi.org yet for some reason): https://pypi.org/project/Flask-COMBO-JSONAPI/1.0.0/

but github release 1.0.2 got released to pypi as 1.0.0. I think you just need to change the version string here: https://github.com/AdCombo/flask-combo-jsonapi/blob/master/setup.py before creating a new release, and the pypi version will be the correct one, or perhaps retag the github release 1.0.2 to 1.0.0.
